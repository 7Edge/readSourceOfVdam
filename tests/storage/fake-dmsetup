#!/bin/sh
set -e

# Run the real dmsetup to validate the arguments, dropping the output.
dmsetup "$@" > /dev/null

# Fake the output
cat $FAKE_STDOUT
