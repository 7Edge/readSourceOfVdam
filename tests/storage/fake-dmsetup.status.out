360014053d0b83eff3d347c48509fc426: 0 104857600 multipath 2 0 1 0 3 2 E 0 1 1 67:16 F 4 0 E 0 1 1 65:240 A 84 0 E 0 1 1 66:64 A 39 0
3600140543cb8d7510d54f058c7b3f7ec: 0 209715200 multipath 2 0 1 0 3 1 A 0 1 1 65:224 A 0 0 E 0 1 1 65:160 A 0 0 E 0 1 1 66:176 F 1 0
